### ISSUE TITLE
Add Partners Section with help tip

### ISSUE THREAD

**Issue by kjaymiller** [BlackPythonDevs/blackpythondevs.github.io#372]
## TLDR

Create a section on the Home Screen and a solo page with sections for each partner. 

---

We have a couple of partners that are giving us affiliate links. 

These are not quite sponsorships but are basically a rate where we get a kickback if people use our codes. 

I would like to add two things. 

The first is a section on the home page encouraging users to select the logo of a partner. When the logo is selected it would go to a dedicated **partnerships** page that explain the partnership. 

Example: 

On the Home Screen, if you select the Training.talkpython.fm logo it will go to blackpythondevs.com/partnerships#training-talkpython which explains that %20 of the cost of the course will be donated to Black Python Devs and the course will be %10 percent off for the user.
